config/routes.rb

```diff
@@ -11,7 +11,7 @@
   get "/account", to: "users#show"
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
-  get "/logout", to: "sessions#destroy"
+  delete "/logout", to: "sessions#destroy"
```

Comment:
This needs to be a GET request; there is no such thing as a DELETE request. I'm surprised this works with putting a DELETE there.